changelog/17.0/17.0.0/summary.md

```diff
@@ -49,6 +50,16 @@ Here is a (condensed) example of stats output:
 }
 ```
 
+
+#### <a id="keyspace-name-validation"> Keyspace name validation in TopoServer
+
+Prior to v17, it was possible to create a keyspace with invalid characters, which would then be inaccessible to various cluster management operations.
+
+Now, the TopoServer's `GetKeyspace` and `CreateKeyspace` methods return an error if given an invalid name.
+
+Keyspace names may no longer contain the following characters:
+- Forward slash ("/").
```

Comment:
It was usable in some ways (basically any way that didn't try to parse an input as both a keyspace _and_ shard, but just a keyspace) so you could get yourself into potentially weird spots. I'll readily admit to not having completely thought through every place this technically would have been usable, since I'm now making it completely unusable anyway.